### PR TITLE
🌐improved translation for ZH_CN

### PR DESCRIPTION
**Pull request by purplegrape**

### CODE REVIEW

**Comment by purplegrape** [comment]
1、translation update
2、fix some missing text
3、adjust the line-number against en.dict  (via blank line)

File: datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php [hunk 1]
```diff
@@ -62,7 +62,7 @@
 	'Class:FAQ/Attribute:error_code+' => '',
 	'Class:FAQ/Attribute:key_words' => '关键字',
 	'Class:FAQ/Attribute:key_words+' => '',
-	'Class:FAQ/Attribute:domains' => 'Domains~~',
+	'Class:FAQ/Attribute:domains' => '领域~~',
```
Comment (Molkobain) [inline datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php:65]:
```suggestion
	'Class:FAQ/Attribute:domains' => '领域',
```

File: dictionaries/zh_cn.dictionary.itop.ui.php [hunk 2]
```diff
@@ -342,49 +347,56 @@
 	'Menu:WelcomeMenuPage+' => '欢迎使用 '.ITOP_APPLICATION_SHORT, // Duplicated into itop-welcome-itil (will be removed from here...)
 	'UI:WelcomeMenu:Title' => '欢迎使用 '.ITOP_APPLICATION_SHORT,
 
-	'UI:WelcomeMenu:LeftBlock' => '<p>iTop 是完全开源的IT 操作门户.</p>
+	'UI:WelcomeMenu:LeftBlock' => '<p>'.ITOP_APPLICATION_SHORT.' 是完全开源的IT 操作门户.</p>
```
Comment (Molkobain) [inline dictionaries/zh_cn.dictionary.itop.ui.php:350]: Thank you! 🙏

**Review by Molkobain — CHANGES_REQUESTED** [review]
Just a small change for an entry where the "~~" should be removed, otherwise it's all good, thank you very much!

File: datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php (hunk 1 above)
Comment (purplegrape) [reply datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php:65]: yes ,I've noticed it.  
it's a temp solution, but I can't think about another word to describe it accurately

Comment (Hipska) [reply datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php:65]: The mark can stay for a temp solution IMHO, I did it as well for Dutch translations (NL-NL).

Comment (purplegrape) [reply datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php:65]: same issue via the translation of “profiles” （line 1674-1676），I add the suffix  ~~ 
hope someone or me  fix it later

Comment (Molkobain) [reply datamodels/2.x/itop-faq-light/dictionaries/zh_cn.dict.itop-faq-light.php:65]: Ok if it is temporary then it makes sense. Thanks!

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Merged, thanks!

File: dictionaries/zh_cn.dictionary.itop.ui.php (hunk 2 above)
Comment (piRGoif) [reply dictionaries/zh_cn.dictionary.itop.ui.php:350]: I don't see this modification in the merge commit (05e98be6) ?

Comment (Hipska) [reply dictionaries/zh_cn.dictionary.itop.ui.php:350]: Correct, see the actual merge commit of this PR: a00f415

Comment (piRGoif) [reply dictionaries/zh_cn.dictionary.itop.ui.php:350]: Woops right, I made a mistake on the merge commit, thanks for the correction Thomas :) Both commits are for different PRs, but both of them don't have their respective PR ref in their message, too bad :/

About using the constant, I asked Guillaume and he said he asked the author to do it, but as it was an extra modification he merged the PR anyway without waiting for this to be done.

I'm closing the conversation

**Pull request merged**